railties/lib/rails/generators/rails/app/templates/config/initializers/hsts_subdomains.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.action_dispatch.hsts_subdomains = true
```

Comment:
@rafaelfranca We can do 

``` ruby
Rails.application.config.ssl_options.merge!({ hsts: { subdomains: true } })
```

But that can mess up with setting the default hsts options here - https://github.com/prathamesh-sonpatki/rails/blob/master/actionpack/lib/action_dispatch/middleware/ssl.rb#L82.

Note that we are passing options only in case of https://github.com/prathamesh-sonpatki/rails/blob/master/actionpack/lib/action_dispatch/middleware/ssl.rb#L92. 

options are `config.ssl_options[:hsts]` 

So if we want to reuse config.ssl_options then we will have to update each branch in https://github.com/prathamesh-sonpatki/rails/blob/master/actionpack/lib/action_dispatch/middleware/ssl.rb#L82 to pass options[:subdomains].

It will override user's config.ssl_options[:hsts] though as we will have a default hash instead of true/false/nil/custom options.